### PR TITLE
Fix broken, fragile and slow tests around circuit breaker

### PR DESCRIPTION
**Pull request by taiki45**
And also

- Introduce RSpec configuration with `rspec --init` so that measure test execution time.
- Ignore `.ruby-version` file.

@cookpad/dev-infra Please take a look.

### CODE REVIEW

File: spec/expeditor/circuit_break_function_spec.rb [hunk 1]
```diff
@@ -83,39 +83,41 @@
     end
 
     context 'with circuit break (large case)' do
-      it 'should be ok' do
+      specify 'circuit will be opened after 100 failure and it skips success_commands' do
```
Comment (eagletmt) [inline spec/expeditor/circuit_break_function_spec.rb]: [nits] "and skip success_commands"

File: spec/expeditor/circuit_break_function_spec.rb [hunk 2]
```diff
@@ -83,39 +83,41 @@
...
+          raise "Won't reach here"
         end.set_fallback do |e|
           reason = e
-          0
+          result
         end
         command.start
-        expect(command.get).to eq(0)
+
+        expect(command.get).to eq(result)
```
Comment (eagletmt) [inline spec/expeditor/circuit_break_function_spec.rb]: Should be compared by `equal?` ?

File: spec/expeditor/service_spec.rb [hunk 3]
```diff
@@ -89,16 +89,18 @@
   end
 
   describe '#status' do
-    let(:service) { Expeditor::Service.new(sleep: 10) }
     it 'returns current status' do
+      # Set large value of period in case test takes long time.
+      service = Expeditor::Service.new(period: 100)
```
Comment (eagletmt) [inline spec/expeditor/service_spec.rb]: Why did you move `service` from `let`?

Comment (taiki45) [inline spec/expeditor/service_spec.rb]: In other test cases, Expeditor uses ordinary variable assignment not using `let`. And since this `service` is not shared by multiple examples, there are no advantage to use `let` here.

File: spec/expeditor/circuit_break_function_spec.rb (hunk 1 above)
Comment (taiki45) [inline spec/expeditor/circuit_break_function_spec.rb]: 👍

File: spec/expeditor/circuit_break_function_spec.rb (hunk 2 above)
Comment (taiki45) [inline spec/expeditor/circuit_break_function_spec.rb]: I think so: 776d84f

File: spec/expeditor/service_spec.rb (hunk 3 above)
Comment (eagletmt) [inline spec/expeditor/service_spec.rb]: > there are no advantage

I prefer `let` syntax to clarify the test target even if it isn't shared, but I don't have so strong opinion on this 🙆

Comment (taiki45) [inline spec/expeditor/service_spec.rb]: >to clarify the test target

Aha, I should take the advantage too, but let me leave these "without `let` code" as they are here.
